Jolt/Physics/Collision/Shape/ScaledShape.cpp

```diff
@@ -178,9 +178,9 @@ bool ScaledShape::IsValidScale(Vec3Arg inScale) const
 
 Vec3 ScaledShape::MakeScaleValid(Vec3Arg inScale) const
 {
-	Vec3 scale = ScaleHelpers::MakeNonZeroScale(inScale);
+	Vec3 scale = ScaleHelpers::MakeNonZeroScale(mScale);
```

Comment:
I think we have to assume that mScale is non zero here. There are other functions that blindly invert the value. Bonus points for adding a check in the constructor and failing shape creation / adding a unit test to verify that this code is correct. (I can do both when I am back from holiday)

Reply:
Makes sense. I'll get it done.

Reply:
I wasn't sure how crazy I should go with the unit tests. Let me know if I should add some more.

Reply:
Thanks! I will probably add some extra unit tests later. For now this is fine.